src/renderers/common/ChainMap.js

```diff
@@ -36,10 +65,18 @@ export default class ChainMap {
 
 		}
 
-		return map.set( keys[ keys.length - 1 ], value );
+		map.set( keys[ keys.length - 1 ], value );
+
+		return this;
```

Comment:
@sunag It seems more consistent to return the reference to the Chain Map itself instead of one the internal weak maps that holds the value.